PlaygroundComponent.jsx

```diff
@@ -32,7 +32,7 @@ export const PlaygroundComponent = () => {
   const history = createBrowserHistory();
 
   useEffect(() => {
-    if (selected === "Select API Call - Version 3") {
+    if (selected === "Select API Call - Version 3" && window.location.href.indexOf('#') > -1 ) {
```

Comment:
What type of issue is this code trying to resolve? From my understanding, we push the location pathname to history when there is a hashtag value active while the placeholder is still active. But, why? (thanks to Huberts observation)

Reply:
To the above. I see in line 35-37 that you introduce condition on selected state (select api call line 30).  But I don't understand, why this string can't just be the placeholder of the dropdown. You introduce the component of dropdown. Pls consider just removing those dependencies on local state and put the "Select API Call - Version 3" as placeholder text to the dropdown you introduce

Reply:
So the code of line 35-37 checks if their is a hash value in the URL so that it removes the schema wrapper component since it does not change on refresh